content/applications/hr/payroll/configuration.rst

```diff
@@ -0,0 +1,356 @@
+
+When creating payslips, it is sometimes necessary to add other entries for specific circumstances,
+like expenses, reimbursements, or deductions. These other inputs can be configured by going to
+:menuselection:`Payroll --> Configuration --> Other Input Types`.
+
+.. image:: configuration/other-input.png
+   :align: center
+   :alt: Other input types for payroll.
+
+To create a new :guilabel:`Input Type`, click the :guilabel:`Create` button. Enter the
+:guilabel:`Description`, the :guilabel:`Code`, and which structure it applies to. Click the
+:guilabel:`Save` button to save the changes, or click :guilabel:`Discard` to delete the entry.
+
+.. image:: configuration/input-type-new.png
+   :align: center
+   :alt: Create a new Input Type.
+
+Salary package configurator
+===========================
+
+The various options under the :guilabel:`Salary Package Configurator` section of the
+:menuselection:`Payroll --> Configuration` menu all affect an employee's potential salary. These
+sections (:guilabel:`Advantages`, :guilabel:`Personal Info`, and :guilabel:`Resume`) specify what
+benefits can be offered to an employee in their salary package.
+
+Depending on what information an employee enters (such as deductions, dependents, etc.), their
+salary is adjusted accordingly. When an applicant applies for a job on the company website, the
+sections under :guilabel:`Salary Package Configurator` directly affect what the applicant sees, and
+what is populated as the applicant enters information.
+
+Advantages
+----------
+
+When offering potential employees a position, there can be certain *advantages* set in Odoo in
+addition to the salary to make the offer more appealing (such as extra time off, access to a company
+car, reimbursement for a phone or internet, etc.). Advantages are grouped by structure type.
+
+To see the advantages, go to :menuselection:`Payroll --> Configuration --> Advantages`.
+
+.. image:: configuration/advantages.png
+   :align: center
+   :alt: Settings available for payroll.
+
+To make a new advantage, click the :guilabel:`Create` button. Enter the information in the fields,
+then click the :guilabel:`Save` button to save the changes, or click :guilabel:`Discard` to delete
+the entry.
+
+.. image:: configuration/new-advantage.png
+   :align: center
+   :alt: List of advantages employee's can have.
+
+The required fields for an advantage are:
+
+ - :guilabel:`Name`: Enter the name for the advantage.
+ - :guilabel:`Advantage Field`: Select from the drop-down menu what field in the payslip this
+   advantage appears under.
+ - :guilabel:`Advantage Type`: Select from the drop-down menu what type of advantage the benefit is.
+   Select from :guilabel:`Monthly Benefit in Kind`, :guilabel:`Monthly Advantages in Net`,
+   :guilabel:`Monthly Advantages in Cash`, or :guilabel:`Yearly Advantages in Cash`.
+ - :guilabel:`Salary Structure Type`: Select from the drop-down menu which salary structure type
+   this advantage applies to.
```

Comment:
These bullet points are indented by a single space, remove it please :)
```suggestion
- :guilabel:`Name`: Enter the name for the advantage.
- :guilabel:`Advantage Field`: Select from the drop-down menu what field in the payslip this
  advantage appears under.
- :guilabel:`Advantage Type`: Select from the drop-down menu what type of advantage the benefit is.
  Select from :guilabel:`Monthly Benefit in Kind`, :guilabel:`Monthly Advantages in Net`,
  :guilabel:`Monthly Advantages in Cash`, or :guilabel:`Yearly Advantages in Cash`.
- :guilabel:`Salary Structure Type`: Select from the drop-down menu which salary structure type this
  advantage applies to.
```

Reply:
Oh no, I thought you needed a space before the bullet list =D fixed that, and will fix that moving forward! (I believe it is in other docs, FYI, I'll try to change them!)